watchdog/watchdog.go

```diff
@@ -24,15 +17,15 @@ var (
 	cancelFunc context.CancelFunc
 )
 
-// Monitor loops over each endpoint and starts a goroutine to monitor each endpoint separately
+// Monitor loops over each endpoint and starts a goroutine to monitorEndpoint each endpoint separately
 func Monitor(cfg *config.Config) {
 	ctx, cancelFunc = context.WithCancel(context.Background())
 	extraLabels := cfg.GetUniqueExtraMetricLabels()
 	for _, endpoint := range cfg.Endpoints {
 		if endpoint.IsEnabled() {
 			// To prevent multiple requests from running at the same time, we'll wait for a little before each iteration
-			time.Sleep(777 * time.Millisecond)
-			go monitor(endpoint, cfg.Alerting, cfg.Maintenance, cfg.Connectivity, cfg.DisableMonitoringLock, cfg.Metrics, extraLabels, ctx)
+			time.Sleep(222 * time.Millisecond)
+			go monitorEndpoint(endpoint, cfg.Alerting, cfg.Maintenance, cfg.Connectivity, cfg.DisableMonitoringLock, cfg.Metrics, extraLabels, ctx)
```

Comment:
Does it really make sense to have this many parameters? We could just pass `endpoint, cfg, extraLabels, ctx`.

Also, to be honest, the extraLabels implementation really doesn't seem like it belongs here.

Reply:
Fixed by [42a7889](https://github.com/TwiN/gatus/pull/1239/commits/42a78897422671c56666700817fc271efabd10dd)